gpu_ocean/SWESimulators/OceanStateNoise.py

```diff
@@ -142,39 +152,37 @@ def fromsim(cls, sim, soar_q0=None, soar_L=None,
         
         
     def getSeed(self):
-        return self.seed.download(self.cl_queue)
+        return self.seed.download(self.gpu_stream)
     
     def resetSeed(self):
+        # Generate seed:
         self.floatMax = 2147483648.0
         self.host_seed = np.random.rand(self.seed_ny, self.seed_nx)*self.floatMax
-        self.host_seed.astype(np.float32, order='C')
-        self.seed.upload(self.cl_queue, self.host_seed)
+        self.host_seed = self.host_seed.astype(np.uint64, order='C')
```

Comment:
Is the type change here intentional?

Reply:
Yepp. The float32 here was actually a bug.